site/snippets/getting-started/setup-app/providers.tsx

```diff
@@ -1,14 +1,25 @@
 "use client";
 
+import { createConfig } from "@alchemy/aa-alchemy/config";
+import { AlchemyAccountProvider } from "@alchemy/aa-alchemy/react";
+import { arbitrumSepolia } from "@alchemy/aa-core";
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { PropsWithChildren, Suspense } from "react";
 
 export const Providers = (props: PropsWithChildren) => {
   const queryClient = new QueryClient();
+  const config = createConfig({
+    rpcUrl: "/api/rpc",
```

Comment:
do we want to explain this? Or point to somewhere in code why we are doing this?

Reply:
yeah good idea, i'll leave a comment